charts/langsmith/docs/DELETE-WORKSPACE.md

```diff
@@ -0,0 +1,66 @@
+# Deleting Traces
+
+The LangSmith does not currently support the deletion of an individual workspace from an organization.  This, however, can be accomplished by directly removing all traces from all materialized views in ClickHouse (except the runs_history views) and the runs table itself and then removing the Workspace from the Postgres tenants table.
```

Comment:
do we also need to cleanup feedback?

Reply:
Hmm...good callout for workspace.  Will add.

Reply:
Ah, feedbacks already in.  Will add to docs.

Reply:
https://github.com/langchain-ai/helm/blob/be4c3e6444df11195a12bff9e963224a65ff75a9/charts/langsmith/scripts/delete_trace_by_id.sh#L249-L251

Reply:
Updated docs to mention feedback.